### PR TITLE
Stack Switcher → SettingsSidebar

### PR DESCRIPTION
**Pull request by danirabbit**

### CODE REVIEW

**Comment by hanaral** [comment]
Something about this layout seems off, maybe it's that the current one is more natural to read (I think because it's above the page and horizontally laid out)

**Comment by jeremypw** [comment]
@danrabbit  I have fixed conflicts with master.  One problem I have noticed with using Granite.SimpleSettingsPage (a scrolled window) with Multi-tasking View is that if the window height is such as to require scrolling then if you scroll on the content area you will at some point enter a combo and start changing the options instead of scrolling - which is definitely undesirable.  The simplest solution is to set a minimum height on the Settings window sufficient that scrolling is never required on Multitasking View.

**Comment by jeremypw** [comment]
I have proposed a change to Switchboard (https://github.com/elementary/switchboard/pull/242) so that this view does not have to scroll and all the settings are visible when it is opened.

**Review by jeremypw — CHANGES_REQUESTED** [review]
The code looks good and the change makes sense in the context of a strategy of moving all the plugs using a Stack to a SettingsSidebar (as many are already).

The screenshots need updating.

**Review by lenemter — COMMENTED** [review]
I get warnings when I close the System Settings app in this branch:
```
** (io.elementary.switchboard:4237): CRITICAL **: 20:14:27.643: granite_settings_sidebar_row_get_page: assertion 'self != NULL' failed

(io.elementary.switchboard:4237): Gtk-CRITICAL **: 20:14:27.643: gtk_stack_set_visible_child: assertion 'GTK_IS_WIDGET (child)' failed
```

Overall I think this is a step in a good direction but it needs more polishing. For example: 
1. 'Wallpaper' and 'Dock & Panel' pages shouldn't use the same icon;
2. Appearance tab looks weird compared to other pages because of it's long description while other pages don't have it at all;
3. IMO It would be nice if every page's max width was the same and these header labels wouldn't jump around

File: src/Views/Appearance.vala [hunk 1]
```diff
@@ -188,15 +188,10 @@ public class PantheonShell.Appearance : Gtk.Box {
...
-            margin_start = 12,
-            margin_end = 12,
-            margin_bottom = 24
+            row_spacing = 6
         };
 
         if (((GLib.DBusProxy) pantheon_act).get_cached_property ("PrefersColorScheme") != null) {
-            grid.attach (dark_label, 0, 0, 2);
-            grid.attach (dark_info, 0, 1, 2);
             grid.attach (prefer_style_box, 0, 2, 2);
```
Comment (lenemter) [inline src/Views/Appearance.vala:195]: This grid's attaches should start with `0, 0`, not from `0, 2` straightaway.

**Comment by 1280px** [comment]
> Something about this layout seems off, maybe it's that the current one is more natural to read (I think because it's above the page and horizontally laid out)

It also sacrifices more horizontal space, yet doesn't resolve the problem of vertical space usage (mainly because of the headers, which don't provide any useful information). Considering the sidebar doesn't provide neither the grouping by categories nor the state, I think it will might also make the uI less consistent?

**Pull request closed without merging**